xclim/data/fr.json

```
@@ -799,6 +799,12 @@
     "long_name": "Indice forêt météo",
     "description": "Évaluation numérique de l'intensité du feu."
   },
+  "DMC": {
+    "long_name": "Indice d'humidité de l'humus (DMC)",
+    "description": "Évaluation numérique de la teneur en eau moyenne des couches organiques peu compactes d'une profondeur modérée.",
+    "title": "Indice d'humidité de l'humus",
+    "abstract": "Évaluation numérique de la teneur en eau moyenne des couches organiques peu compactes d'une profondeur modérée."
```

```suggestion
    "long_name": "Indice de l'humus",
    "description": "Évaluation numérique de la teneur moyenne en eau des couches organiques peu tassées de moyenne épaisseur.",
    "title": "Indice de l'humus (IH)",
    "abstract": "Évaluation numérique de la teneur moyenne en eau des couches organiques peu tassées de moyenne épaisseur."
```
This is what the FWI indicator puts for DMC, and it comes from an official document. I liked yours better though...
And I just realize we don't have the correct long name for the FWI one :roll_eyes: .

https://cwfis.cfs.nrcan.gc.ca/renseignements/sommaire/fwi